### ISSUE TITLE
Showing an AFArray does not work in Julia v0.5

### ISSUE THREAD

**Issue by dpsanders** [JuliaComputing/ArrayFire.jl#74]
```
MethodError: no method matching print_matrix(::IOContext{Base.Terminals.TTYTerminal}, ::ArrayFire.AFArray{Float64,2}, ::String, ::String, ::String)
```

**Comment by dpsanders** [comment]
This is fixed in master.

**Comment by ViralBShah** [comment]
Do say if a new release should be made in case needed for class.

**Comment by dpsanders** [comment]
That would be great, yes please.

**Comment by ViralBShah** [comment]
Maybe tomorrow and hopefully with the `size` fix.

**Comment by lilinjn** [comment]
hi -- a new release may be a good idea to avoid panic with new users -- I also just stumbled on this in my first interaction with ArrayFire.jl after Pkg.add().  cheers, nehal

**Comment by ranjanan** [comment]
@lilinjn thanks for the suggestion. I'm considering tagging a new release for bug fixes for v0.5 soon.
